tests/solids/diamondC_2x1x1-Gaussian_pp_cplx_MSD_excited/CMakeLists.txt

```diff
@@ -0,0 +1,156 @@
+# 
+#qmc_ref_ground_long_noj  series 0 
+#  LocalEnergy           =         -21.36523 +/-          0.00013 
+#  Variance              =            4.1743 +/-           0.0045 
+#  Kinetic               =          19.75794 +/-          0.00042 
+#  LocalPotential        =         -41.12317 +/-          0.00044 
+#  ElecElec              =          -4.60299 +/-          0.00016 
+#  LocalECP              =         -13.07070 +/-          0.00056 
+#  NonLocalECP           =           2.10185 +/-          0.00019 
+#  IonIon                =            -25.55 +/-             0.00 
+#  LocalEnergy_sq        =          460.6475 +/-           0.0069 
+#  BlockWeight           =        1310720.00 +/-             0.00 
+#  BlockCPU              =          22.54826 +/-          0.00013 
+#  AcceptRatio           =         0.9355798 +/-        0.0000023 
+#  Efficiency            =             23.55 +/-             0.00 
+#  TotalTime             =           6471.35 +/-             0.00 
+#  TotalSamples          =         376176640 +/-                0 
+
+
+
+
+
+
+# * State            2
+# < S^2 >         =   6.026072064414621E-005
+# E               =   -20.7021278060674     
+# Variance        =    1.05958506927153       +/-   0.000000000000000E+000
+# PT norm         =    1.56509030954092       +/-   0.000000000000000E+000
+## PT2             =  -0.488098362967976       +/-   0.000000000000000E+000
+# rPT2            =  -0.141497978456392       +/-   0.000000000000000E+000
+# E+PT2           =   -21.1912261690353       +/-   0.000000000000000E+000
+# E+rPT2          =   -20.8446257845238       +/-   0.000000000000000E+000
+
+#qmc_ref_excited_long  series 0 
+#  LocalEnergy           =         -20.70253 +/-          0.00016 
+#  Variance              =             4.154 +/-            0.012 
+#  Kinetic               =          20.63096 +/-          0.00043 
+#  LocalPotential        =         -41.33350 +/-          0.00044 
+#  ElecElec              =          -4.67055 +/-          0.00018 
+#  LocalECP              =         -13.32539 +/-          0.00057 
+#  NonLocalECP           =           2.21377 +/-          0.00022 
+#  IonIon                =            -25.55 +/-             0.00 
+#  LocalEnergy_sq        =           432.748 +/-            0.013 
+#  BlockWeight           =        1310720.00 +/-             0.00 
+#  BlockCPU              =          22.36911 +/-          0.00013 
+#  AcceptRatio           =         0.9336608 +/-        0.0000020 
+#  Efficiency            =             16.86 +/-             0.00 
+#  TotalTime             =           6487.04 +/-             0.00 
+#  TotalSamples          =         380108800 +/-                0 
+
+      
+
+#Data generated using utils/make_ref_data.sh  diamondC_2x1x1-Gaussian_pp_cplx_MSD_GROUND_SHORT vmc_short_ground.s000.scalar.dat  qmc_ref_ground_long_noj.s000.scalar.dat
+LIST(APPEND diamondC_2x1x1-Gaussian_pp_cplx_MSD_GROUND_SHORT_SCALARS "totenergy" "-21.365243430795 0.068654689630")
```

Comment:
Either use QP reference or add explanation in text.

Reply:
Done